raiden/ui/cli.py

```diff
@@ -310,10 +310,15 @@ def options(func):
                 "--log-file",
                 help="file path for logging to file",
                 default=None,
-                type=str,
+                type=click.Path(dir_okay=False, writable=True, resolve_path=True),
                 show_default=True,
             ),
             option("--log-json", help="Output log lines in JSON format", is_flag=True),
+            option(
+                "--debug-logfile-name",
```

Comment:
Should we hide this by default? I guess users shouldn't touch this.

Reply:
why not? if they can use `--disable-debug-logfile` I don't see why they can't move it around.